### DISCUSSION TITLE
How do I disable the client auto banning

### DISCUSSION THREAD

**Discussion by a8underscore** [c0re100/qBittorrent-Enhanced-Edition#335] [Q&A] [answered]
I use want this client for the appimage, as my seedbox provider only provides outdated qbittorrent & i cant find a appimage or static build of a modern qbittorrent version

but I dont want to ban any clients, even if they are leacher clients, i started the appimage on my machine locally, and I see the options for "auto ban unknown peer from china" and "auto ban bittorrent media player peer", but I dont see the "Auto Ban Xunlei, QQ, Baidu, etc"

**Comment by c0re100** [comment, marked as answer]
I guess you need this.
https://github.com/userdocs/qbittorrent-nox-static/releases/tag/release-4.4.1_v2.0.5

**Reply by a8underscore** [reply]
oh thats great thank you
